custom-elements/throw-on-dynamic-markup-insertion-counter-construct.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+ <html>
+ <head>
+ <title>Custom Elements: create an element for a token must increment and decrement document's throw-on-dynamic-markup-insertion counter</title>
+ <meta name="author" title="Ryosuke Niwa" href="mailto:rniwa@webkit.org">
+ <meta name="assert" content="Invoking document.open, document.write, document.writeln, and document.write must throw an exception when the HTML parser is creating a custom element for a token">
+ <meta name="help" content="https://html.spec.whatwg.org/multipage/parsing.html#create-an-element-for-the-token">
+ <meta name="help" content="https://html.spec.whatwg.org/multipage/dynamic-markup-insertion.html#throw-on-dynamic-markup-insertion-counter">
+ <script src="/resources/testharness.js"></script>
+ <script src="/resources/testharnessreport.js"></script>
+ <script src="/resources/custom-elements-helpers.js"></script>
```

Comment:
This path is invalid... Could you make sure these tests actually work before uploading the next patch?

Reply:
I have run the tests after my last commit(on chrome), they are passing.

Reply:
resources/custom-elements-helper.js is in /custom-elements/ not /

Reply:
I did wpt serve and run the URL of two tests in browsers.

Reply:
I just noticed that @rniwa 's comment had not been fixed. Not sure why the tests still work...

Reply:
I have made the chages in this PR https://github.com/web-platform-tests/wpt/pull/14221/commits